crypto/keys/keyerror/errors.go

```diff
@@ -0,0 +1,81 @@
+package keyerror
```

Comment:
since it's the only file on the folder, lets move this `error.go` file to the `crypto/keys` folder

Reply:
That's will cause cycle dependency. Because `crypto/keys/mintkey/mintkey.go` will depend on this package.

Reply:
oh ok